armi/materials/b4c.py

```diff
@@ -146,10 +159,15 @@ def setDefaultMassFracs(self) -> None:
         self.theoreticalDensityFrac = self.DEFAULT_THEORETICAL_DENSITY_FRAC  # normally is around 0.88-93.
 
     @staticmethod
-    def getMassEnrichmentFromNumEnrich(b10NumFrac: float) -> float:
+    def getMassEnrichmentFromNumEnrich(
+        b10NumFrac: float, b10AtomicMass: float = None, b11AtomicMass: float = None
+    ) -> float:
         """Given a B10 number fraction, give the B10 weight fraction."""
-        b10AtomicMass = nuclideBases.byName["B10"].weight
-        b11AtomicMass = nuclideBases.byName["B11"].weight
+        if b10AtomicMass is None:
+            b10AtomicMass = 10.01293728
+        if b11AtomicMass is None:
+            b11AtomicMass = 11.0093054803
```

Comment:
Oh wow you caught some really silly code here! (Cause even if someone passed it in, it was getting overwritten.)